commands.js

```diff
@@ -1779,6 +1786,7 @@ var commands = exports.commands = {
 		}
 		return;
 	},
+	memoryusagehelp: ["/memoryusage - Shows the amount of memory the server is using. Requires: Console Access"],
```

Comment:
It should be simplified to just returning the contents of `process.memoryUsage()`